scss/components/input-group.scss

```diff
@@ -135,7 +135,13 @@ $block: #{$fd-namespace}-input-group;
         &--button {
           padding: 0;
           [class*="fd-button"] {
```

Comment:
You don't need to write attribute-matching selectors for the `class` attribute.  You can just use `&.fd-button`.

Reply:
@greg-a-smith The proper styling is not applied when using `&.fd-button` as you can see on the following screenshot (incorrect border radius). 

<img width="539" alt="Screen Shot 2019-06-25 at 4 00 20 PM" src="https://user-images.githubusercontent.com/25252963/60129211-7a8a8b80-9762-11e9-8d5f-f1a13f33b1b5.png">

Reply:
Okay, you can leave this for now.  I have created #138 to go through and refactor blocks like this.